crypto/src/main/kotlin/net/corda/v5/crypto/NodeAndWeight.kt

```diff
@@ -0,0 +1,5 @@
+package net.corda.v5.crypto
+
+import java.security.PublicKey
+
+data class NodeAndWeight(val node: PublicKey, val weight: Int = 1)
```

Comment:
Name it `CompositeKeyNodeAndWeight` as when it's standalone class the name as now doesn't convey too much

Reply:
And I think that class can be moved to runtme-os (to crypto-core or maybe even crypto-impl) as it's not being used by any publicly exposed classes/interface - it became internal implementation details

Reply:
done